ingests/wb/latest/wb_gender.py

```diff
@@ -0,0 +1,95 @@
+"""This script has been autogenerated by `walkthrough walden`."""
```

Comment:
I'd suggest to not use "latest" as the version. Currently, you think that this script will always work. But the most likely scenario is that, either in the next update or in a future one, things will change. And then "latest" will need to be modified, and will no longer be "latest". So I'd suggest to just use an actual version (i.e. a date). The worst that can happen is that you'll have to copy/paste the code again once for each version. But we have already assumed this for etl, that's the expected behaviour and it makes sense. Versions are independent pieces of code. This would be happening in git anyway, but we make the changes in versions more explicit by keeping the code of each version at any point in the repos.

Reply:
If you want to keep the logic you have created here, but actually creating new step files, you could do that inside `etl/scripts/`. That's what I did for FAOSTAT with `etl/scripts/faostat/create_new_steps.py`.

Reply:
Sorry, I have mixed here the etl logic with walden logic. Give me a minute to rethink this...

Reply:
I think it makes sense to have a `latest`, given that, if the source changes their logic, then our code will not work anymore, so then it makes sense to overwrite the ingest code, and therefore it will always be the `latest`.